public/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="main.css">
+    <title>Echo</title>
+  </head>
+  <body>
+    <div id="desktop-background" >
+      <div id="d3-canvas">
+
```

Comment:
Hmm, are you planning on using a canvas element?

A canvas is a html5 thing that I expect you will not use. The name is misleading